interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -813,26 +843,24 @@ function postcalendar_admin_testSystem()
         }
     }
     if (strlen($error) > 0) {
-        $info .= "<br /><div style=\"color: red;\">$error</div>";
+        $info .= "<br /><div class='text-danger'>$error</div>";
     }
     array_push($infos, array('smarty compile dir',  $info));
 
     $header = <<<EOF
-	<html>
+	<h1>
 	<head></head>
-	<body bgcolor=
+	<body>
 EOF;
-    $header .= '"' . attr($GLOBALS['style']['BGCOLOR2']) . '">';
     $output .= $header;
-    $output  = postcalendar_adminmenu();
-    $output .= '<table border="1" cellpadding="3" cellspacing="1">';
-    $output .= '  <tr><th align="left">' . xlt('Name') . '</th><th align="left">' . xlt('Value') . '</th>';
-    $output .= '</tr>';
+    $output  = postcalendar_adminmenu("testSystem");
+    $output .= '<div class="container"><table class="table table-bordered table-striped"><thead>';
```

Comment:
This table should have `<div class="table-responsive">` wrapped around it. [See here for further documentation](https://getbootstrap.com/docs/4.4/content/tables/#responsive-tables).